core/src/main/scala/kafka/admin/LeaderElectionCommand.scala

```diff
@@ -283,6 +284,14 @@ private final class LeaderElectionCommandOptions(args: Array[String]) extends Co
     .describedAs("election type")
     .withValuesConvertedBy(ElectionTypeConverter)
 
+  val requestTimeout = parser
+    .accepts(
+      "timeout",
+      CommonClientConfigs.REQUEST_TIMEOUT_MS_DOC)
```

Comment:
This configuration is available in AdminClientConfig `AdminClientConfig.REQUEST_TIMEOUT_MS_DOC`. Can we not introduce CommonClientConfigs as we prefer clean code?

Reply:
Thank you very much! AdminClientConfig REQUEST_TIMEOUT_MS_DOC is a private property. I will rewrite the instructions